### ISSUE TITLE
Color blind friendly icon?

### ISSUE THREAD

**Issue by ascendbruce** [maximelebreton/quick-javascript-switcher#37]
Hi @maximelebreton, just saw [your crowdfunding campaign](https://www.kickstarter.com/projects/376707762/quick-javascript-switcher-chrome-extension-next-re), supported immediately. Thanks for making this super useful extension.

However, I wonder if you consider adding (maybe configurable?) on/off labels in the toggle icon?

Example: iPhone's Settings app > General > Accessibility > On/Off Labels

![img_0658](https://user-images.githubusercontent.com/356911/50484060-89012000-0a29-11e9-9930-a70ff2817d35.PNG)

**Comment by maximelebreton** [comment]
@ascendbruce yep, it's possible with a badge (on/off) text on the browser action (https://developer.chrome.com/extensions/browserAction#method-setBadgeText)

would this render be suitable?

**Comment by maximelebreton** [comment]
And just for asking, I suppose the toggle position is not enough?
https://github.com/maximelebreton/quick-javascript-switcher/issues/1

**Comment by jamiesonbecker** [comment]
In the current icon, the color doesn't just change -- it also moves from the top left to bottom left. It's not extremely obvious, but it works. As long as you do something like that in the new icon (changing the shape or layout of the toggle/icon), it should be good.
